BigQueryProvider/BigQueryConnection.cs

```diff
@@ -29,16 +29,7 @@ public class BigQueryConnection : DbConnection, ICloneable {
 
         public override void ChangeDatabase(string databaseName) {
             CheckDisposed();
```

Comment:
According to MSDN, the behavior in the case of a closed connection is not documented, so I believe there's no harm in doing this way. Moreover, before this change, the current database used to be changed in just the same way (with the only difference that the connection used to be opened).